plutus-tx/src/PlutusTx/Builtins/HasOpaque.hs

```diff
@@ -1,15 +1,17 @@
 {-# LANGUAGE CPP                      #-}
 {-# LANGUAGE DefaultSignatures        #-}
+{-# LANGUAGE DerivingStrategies       #-}
 {-# LANGUAGE FlexibleInstances        #-}
 {-# LANGUAGE FunctionalDependencies   #-}
 {-# LANGUAGE StandaloneKindSignatures #-}
 {-# LANGUAGE TypeFamilies             #-}
 {-# LANGUAGE TypeOperators            #-}
 {-# LANGUAGE UndecidableInstances     #-}
+
 {-# OPTIONS_GHC -Wno-orphans #-}
 {-# OPTIONS_GHC -fno-omit-interface-pragmas #-}
 {-# OPTIONS_GHC -fno-specialise #-}
-{-# LANGUAGE DerivingStrategies       #-}
+{-# OPTIONS_GHC -Wwarn=unrecognised-pragmas #-}
```

Comment:
Maybe wrap this in `#if !MIN_VERSION_base(4, 15, 0)` here and in other modules that already use `LANGUAGE CPP`.

That way, it's easier to spot what to delete when we drop support for ghc 8. (It could probably be automated with a `sed` script, in fact.)

Reply:
Done, thanks.

Reply:
(for all modules, not just those where we have `CPP` already, I added it where it was required)